.github/unittest.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Prepare conda
+CONDA_PATH=$(which conda)
+eval "$(${CONDA_PATH} shell.bash hook)"
+
+echo '::group::Set PyTorch conda channel and wheel index'
+# TODO: Can we maybe have this as environment variable in the job template? For example, `IS_RELEASE`.
+if [[ (${GITHUB_EVENT_NAME} = 'pull_request' && (${GITHUB_BASE_REF} = 'release'*)) || (${GITHUB_REF} = 'refs/heads/release'*) ]]; then
+  CHANNEL_ID=test
+else
+  CHANNEL_ID=nightly
+fi
+PYTORCH_CONDA_CHANNEL=pytorch-"${CHANNEL_ID}"
+echo "PYTORCH_CONDA_CHANNEL=${PYTORCH_CONDA_CHANNEL}"
+
+case $GPU_ARCH_TYPE in
+  cpu)
+    GPU_ARCH_ID="cpu"
+    ;;
+  cuda)
+    VERSION_WITHOUT_DOT=$(echo "${GPU_ARCH_VERSION}" | sed 's/\.//')
+    GPU_ARCH_ID="cu${VERSION_WITHOUT_DOT}"
+    ;;
+  *)
+    echo "Unknown GPU_ARCH_TYPE=${GPU_ARCH_TYPE}"
+    exit 1
+    ;;
+esac
+PYTORCH_WHEEL_INDEX="https://download.pytorch.org/whl/${CHANNEL_ID}/${GPU_ARCH_ID}"
+echo "PYTORCH_WHEEL_INDEX=${PYTORCH_WHEEL_INDEX}"
+echo '::endgroup::'
+
+echo '::group::Create build environment'
+conda create \
+  --name ci \
+  --quiet --yes \
+  python="${PYTHON_VERSION}" pip \
+  ninja libpng jpeg 'ffmpeg<4.3' \
+  -c "${PYTORCH_CONDA_CHANNEL}" \
+  -c conda-forge
```

Comment:
@malfet as discussed offline, we now pull all non-Python dependencies from our channels first and only afterwards fall back to conda-forge. However, currently none of our dependencies are available from `pytorch-nightly` or `pytorch-test`. Weirdly enough, `ffmpeg` is available int the stable `pytorch` channel. Meaning, we currently pull everything from `conda-forge`:

https://github.com/pytorch/vision/actions/runs/4231160078/jobs/7349334143#step:10:134

Reply:
After some more offline discussion, I've reverted this to use the `defaults` channel as "fallback" again in 8e10b25b71be0201e018e7ee6b4492b160959fed.